### ISSUE TITLE
numpy to feed in network

### ISSUE THREAD

**Issue by NomiMalik0207** [arundasan91/DeXpression#3]
Cannot feed value of shape (350,) for Tensor 'TargetsData/Y:0', which has shape '(?, 7)'
when i am trying to feed numpy into network is give the above error? how i can solve this.
thanks

**Comment by arundasan91** [comment]
Your output label file is not of proper shape. Required shape is `(num_images, 7)` where `num_images` are the number of images in training set.

Please check the shape of your output y values by using for example `variable_y.shape`

**Comment by NomiMalik0207** [comment]
yup.. its shape is (1304,1)  i can reshape it using onehot encoding to (1304,8) but when i try to change (1304,7) it give the error of index 7 is out of bounds for axis 0 with size 7. this have you anyother script of changing dataset in to numpy? if yes than please share the link it will be a greatness of your's .
thank you

**Comment by arundasan91** [comment]
I see that your numpy file has 8 labels. Please change the final softmax layer of the network to reflect that. You could just change https://github.com/arundasan91/DeXpression/blob/master/model_deXpression.py#L26 and try again.
